sme_finance_application_schema/translations.py

```diff
@@ -1,6 +1,6 @@
 import re
 
-def finance_application_v3_to_sme_contact_v3(finance_application):
+def finance_application_v3_to_sme_contact_v3(finance_application, backfill_value='Unknown'):
```

Comment:
I don't think this should be a param as 'Unknown' is currently a static value in the code.

Reply:
*nod*